src/Components/FrisbeeFilters.test.js

```diff
@@ -36,26 +39,33 @@ describe('<FrisbeeFilters />', () => {
       const DummyScreen = props => null;
       const Stack = createStackNavigator();
 
+      const mockStore = configureMockStore()({
```

Comment:
I did something different in DrillPage.test.js (I guess I had the same problem)

If it is cleaner to use this package, maybe we could use it everywhere?